tests/test_tools.py

```diff
@@ -373,7 +373,20 @@ def test_get_conflicts(self, client, helpers):
             }
         )
 
-        conflicts = openreview.tools.get_conflicts([profile1], profile2)
+        intern_profile = openreview.Profile(
+            id='Test_Conflict3',
+            content={
+                'emails': ['user3@345.com'],
+                'history': [{
+                    'position': 'Intern',
+                    'institution': {
+                        'domain': 'user3@cmu.edu'
+                    }
+                }]
+            }
+        )
+
+        conflicts = openreview.tools.get_conflicts([profile1, intern_profile], profile2)
```

Comment:
if the position is "intern" then it shouldn't be a profile, why the test is asserting "cmu.edu"?

Reply:
Fixed it, it wasn't even testing the intern code as policy was not set to `neurips`.